packages/inventory/src/components/detail/ApplicationDetails.js

```diff
@@ -38,6 +42,11 @@ const ApplicationDetails = ({ onTabSelect, ...props }) => {
 };
 
 ApplicationDetails.propTypes = {
+    appList: PropTypes.arrayOf(PropTypes.shape({
+        title: PropTypes.node,
+        name: PropTypes.string,
```

Comment:
Name seems to be required. Otherwise, the find inside onSelect callback will nor provide correct results.